tools/cfgs/argo2_models/cbgs_voxel01_voxelnext.yaml

```diff
@@ -4,7 +4,7 @@ CLASS_NAMES: ['Regular_vehicle', 'Pedestrian', 'Bicyclist', 'Motorcyclist', 'Whe
             'Message_board_trailer', 'Bicycle', 'Motorcycle', 'Wheeled_device', 'Wheelchair', 'Stroller', 'Dog']
 
 DATA_CONFIG:
-    _BASE_CONFIG_: cfgs/dataset_configs/argo2_dataset.yaml
+    _BASE_CONFIG_: /OpenPCDet/tools/cfgs/dataset_configs/argo2_dataset.yaml
```

Comment:
Note: the paths within this config were incomplete, and I found that full absolute paths were required in order to run the container.